bin/fuel-core/src/cli/run/graphql.rs

```diff
@@ -66,6 +66,26 @@ pub struct GraphQLArgs {
     #[clap(long = "api-request-timeout", default_value = "30s", env)]
     pub api_request_timeout: humantime::Duration,
 
+    /// Maximum allowed block lag for GraphQL fuel block height requests.
+    /// The client waits for the node to catch up if it's behind by no more blocks than
+    /// this tolerance.
+    #[clap(
+        long = "graphql-required-block-height-tolerance",
+        default_value = "10",
+        env
+    )]
+    pub required_fuel_block_height_tolerance: u32,
+
+    /// The minimum time that the node will wait to catch up to the required block height
+    /// of a graphql request. The actual time may be longer and proportional to how
+    /// much the node is behind the required block height.
+    #[clap(
+        long = "graphql-required-block-height-min-timeout-seconds",
+        default_value = "10",
+        env
+    )]
+    pub required_fuel_block_height_min_timeout: u32,
```

Comment:
I think this CLI argument is overkill, it is very hard to understand what it is doing, if you don't look into the code. And even then, it is very hard to trigger this timeout